back/engines/commercial/public_api/app/controllers/public_api/v2/file_attachments_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
```

Comment:
I chose to be a bit safe and only do the `index` and `show` for now, as that's all the ticket requires and so we don't add anything that could become an issue for backwards compatibility in the future if we change the API.

If we want additional endpoints to create or update file attachments, I think we can add those later.